packages/firebase_data_connect/firebase_data_connect/lib/src/firebase_data_connect.dart

```diff
@@ -61,17 +61,11 @@ class FirebaseDataConnect extends FirebasePluginPlatform {
   QueryRef<Data, Variables> query<Data, Variables>(
       String operationName,
       Deserializer<Data> dataDeserializer,
-      Serializer<Variables>? varsSerializer,
+      Serializer<Variables> varsSerializer,
```

Comment:
This is technically a breaking change, but as this package is in preview, I believe the package should be bumped a minor version (i.e. the "x" in this version `0.x.0`).

To bump this version, you need to change your PR title to "feat(fdc)!: make serializer a required argument"